fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/VolatilityTableSelectionSpec.groovy

```diff
@@ -133,6 +133,11 @@ class VolatilityTableSelectionSpec extends BaseDataServletComponentSpec {
                 }
             ],
             "meta": {
+                "schema": {
+                    "limbs": {
+                        "type":"number"
+                    }
+                },
```

Comment:
can we enhance one of these test metrics to use the type, subtype and type metadata?
Maybe we can just change the metric for one test to minimize additional work?